elifetools/utils.py

```diff
@@ -1,8 +1,16 @@
-import cgi
-import htmlentitydefs
 import time
 import calendar
 import re
+from six import iteritems
+
+
+def unicode_value(value):
```

Comment:
could be replaced with `six.u`

Reply:
I did try `six.u` but some of the tests had issues with back slashes vs escape sequences when run through the function, they were not being output as expected. As the `unicode_value` utility function was created previously on the project and worked, I did not investigate it any further. Though I would prefer to have used `six.u`.

Reply:
Okay, good to know. Actually looking at the code and reading the description again makes it kind of clear that it does do something a bit different:
https://github.com/benjaminp/six/blob/master/six.py#L652-L653
(Which I think I used wrongly in some instances)

I think the equivalent to your method is `six.text_type` which is `str` for Python 3 and `unicode` for Python 2. (I now remember I have used that too)

Reply:
Ah, that's sound more like it. Thank you.